.changeset/curvy-insects-design.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/admin.users.v1": patch
+"@wso2is/i18n": patch
+---
```

Comment:
Let's bump console too, otherwise users.v1 dependency in console will not be updated

Reply:
Fixed with cb0ee2cf165319ca096aa44d358799060f08bbf9